alphaplot/src/2Dplot/Legend2D.cpp

```diff
@@ -238,6 +238,33 @@ bool Legend2D::load(XmlStreamReader *xmlreader) {
         xmlreader->raiseWarning(
             tr("Legend2D left margin property setting error"));
     }
+    // legend title
+    while (!xmlreader->atEnd()) {
```

Comment:
probably such a loop is causing the issue bcoz of the legend tag. try manually editing out the 2nd legend block from .aproj file and opening it with the previous version. this block should not be named legend but something else may be legendtitle. doing that would not solve the issue though. if you try to open that file in the previous version, it will get caught up in such a while loop and hang.

Reply:
I realized this in the tests that I did just now. This tag was added to the commit [63040fa](https://github.com/narunlifescience/AlphaPlot/commit/63040faacb89c6cd9bb2983ad4bbe20a879cb81f).
I prefer to delete this part of the code for now. I'm still learning Qt and C++ and it is a little complicated to deal with this issue now.

Reply:
Actually, this is not an issue with your code. the way I implemented this is not right. I will try to think of a better solution for this issue.